packages/frontend/core/src/components/root-app-sidebar/journal-button.tsx

```diff
@@ -34,12 +34,14 @@ export const AppSidebarJournalButton = ({
     [openToday]
   );
 
-  const JournalIcon = useLiveData(
-    docDisplayMetaService.icon$(docCollection.id, {
-      compareDate: new Date(),
-    })
-  );
-  const Icon = isJournal ? JournalIcon : TodayIcon;
+  const Icon =
+    isJournal && journalDate
```

Comment:
This logic should included in `docDisplayMetaService.icon$`, the issue here should be wrong parameter. Should replace `docCollection.id` with `location.pathname.split('/')[1]`(docId)